go.mod

```diff
@@ -8,7 +8,7 @@ replace (
 	// here:
 	github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.3
 	github.com/btcsuite/btcd/v2 => github.com/btcsuite/btcd v0.23.4
-	github.com/checksum0/go-electrum => github.com/keep-network/go-electrum v0.0.0-20230516144526-567f2c1de4bc
+	github.com/checksum0/go-electrum => github.com/keep-network/go-electrum v0.0.0-20230519144954-5d10dfb01e18
```

Comment:
Let's remember about updating to the latest commit from https://github.com/keep-network/go-electrum/pull/2

Reply:
cd684e655146a095b26d19870985b3387d257d9b